setup.py

```diff
@@ -103,6 +103,7 @@
         "black",
         "isort",
         "music21",
+        "miditoolkit",
```

Comment:
Since the two are not directly related to testing functions, how about moving them to `tools/`installers` and `Makefile`. Then default add them in `ci/install.sh`. This ensures that the CI test will have the two packages.